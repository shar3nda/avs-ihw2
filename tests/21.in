wBmUOWvo
3=uy(cOSj?O"te#i^H8@	=Tj\@Ou+q:aL:(25#;=IEYg(1lr>Q4=&^A(sC1JMS;