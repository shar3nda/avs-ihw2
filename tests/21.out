Lowercase: 874, uppercase: 931